error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/IdentityConversionTest.java

```diff
@@ -157,6 +163,11 @@ void identification() {
             "    Mono<Integer> m1 = Mono.from(Mono.just(1));",
             "    // BUG: Diagnostic contains:",
             "    Mono<Integer> m2 = Mono.fromDirect(Mono.just(1));",
+            "",
+            "    // BUG: Diagnostic contains:",
+            "    Matcher matcher1 = Matchers.allOf(annotations(AT_LEAST_ONE, isType(\"foo\")));",
+            "    // BUG: Diagnostic contains:",
+            "    Matcher matcher2 = Matchers.anyOf(annotations(AT_LEAST_ONE, isType(\"bar\")));",
```

Comment:
We can also move these up a bit, in line with the main code order. (Edit: seems there are other inconsistencies. Will move some stuff around.)

Next to that:
1. We can use simpler expressions for this test.
2. We should do this more generally, but since it's cheap for this case: let's also add some non-matching cases.